### PR TITLE
Add a couple use cases of useProjectionsForRegion

### PR DESCRIPTION
**Pull request by ghop02**
Added a hook for getting the projection from a region object + a couple of example uses of it.  Need to clean up some of the comments in model.ts, but the structure should be good to go!

### CODE REVIEW

File: src/screens/internal/ShareImage/ChartShareImage.tsx [hunk 1]
```diff
@@ -12,29 +12,24 @@ import {
...
-import { Projections } from 'common/models/Projections';
 import { MetricChart } from '../../../components/Charts';
 import { ALL_METRICS, getMetricNameExtended } from 'common/metric';
 import { Metric } from 'common/metric';
-import { findCountyByFips } from 'common/locations';
-import { useProjections } from 'common/utils/model';
-import { Projection } from 'common/models/Projection';
+import { useProjectionsFromRegion } from 'common/utils/model';
 import { SCREENSHOT_CLASS } from 'components/Screenshot';
+import { useRegionFromParams } from 'common/regions/region_hooks';
```
Comment (pnavarrc) [inline src/screens/internal/ShareImage/ChartShareImage.tsx:20]: It's better to import from `common/regions`, that way we don't leak implementation details of the module (if we decide to move the function, for example)

File: src/screens/internal/ShareImage/ShareCardImage.tsx [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 import {
   ShareCardWrapper,
   TitleWrapper,
   LastUpdatedWrapper,
 } from './ShareCardImage.style';
 import { DarkScreenshotWrapper } from './ShareImage.style';
 import { formatLocalDate } from 'common/utils';
-import { findCountyByFips } from 'common/locations';
 import { ScreenshotReady, SCREENSHOT_CLASS } from 'components/Screenshot';
+import { useRegionFromParams } from 'common/regions/region_hooks';
```
Comment (pnavarrc) [inline src/screens/internal/ShareImage/ShareCardImage.tsx:12]: same here

File: src/screens/internal/ShareImage/ShareCardImage.tsx [hunk 3]
```diff
@@ -50,21 +44,24 @@ const Header = (props: { isHomePage?: Boolean }) => {
...
-  const [countyOption] = useState(
-    countyFipsId && findCountyByFips(countyFipsId),
-  );
-  stateId = stateId || countyOption.state_code;
-  projections = useProjections(stateId!, countyOption) as any;
+interface LocationShareCardProps {
+  region: Region;
+}
+
+const LocationShareCard = ({ region }: LocationShareCardProps) => {
```
Comment (pnavarrc) [inline src/screens/internal/ShareImage/ShareCardImage.tsx:63]: Maybe this component should use a projection instead? It doesn't seem that the component uses the region for anything other than getting the projections. We can do that later

**Review by pnavarrc — APPROVED** [review]
Added some notes, looks good otherwise!

File: src/screens/internal/ShareImage/ShareCardImage.tsx (hunk 3 above)
Comment (ghop02) [reply src/screens/internal/ShareImage/ShareCardImage.tsx:63]: Yeah I think ideally it would, probably will keep it as is for now

**Pull request merged**